Emojis 13.1 support

jefersonbitencourtv opened the pull request.

jefersonbitencourtv committed:
Update emojis.json

jefersonbitencourtv committed:
Emojis 13.1 characters

Emojis 13.1 characters

jefersonbitencourtv committed:
Emoji 13.1 characters

jefersonbitencourtv commented:
@vdurmont I believe this is interesting for people, these changes worked perfectly here

jefersonbitencourtv committed:
Update emojis.json

Detective bug adjusted

jefersonbitencourtv committed:
Update emojis.json

Update

rogeriofonseca commented:
Hello,
Perhaps your branch is failing tests.
`1 Error and 8 Failres tests.`

**For example:**
getForAlias_returns_the_emoji_for_the_alias(com.vdurmont.emoji.EmojiManagerTest)  <<< FAILURE!

**expected:**<[smiling face with open mouth and] smiling eyes> **but was:**<[grinning face with] smiling eyes>
        at com.vdurmont.emoji.EmojiManagerTest.getForAlias_returns_the_emoji_for_the_alias(EmojiManagerTest.java:78)



Please, check again.

jefersonbitencourtv committed:
Some emojis were behaving incorrectly with the use of skin tone coming from whats.

Emoji :junction :foi necessário entre os alias para funcionar o tom de pele, alguns emojis possuem use_for_tone em seu final para uso funcionamento correto

jefersonbitencourtv committed:
Emoji: junction: it was necessary between aliases to work the skin tone, some emojis were replicated with use_for_tone on their end for correct use

Emoji: junction: it was necessary between aliases to work the skin tone, some emojis were replicated with use_for_tone on their end for correct use

jefersonbitencourtv committed:
Update tests

Some tests caused an error due to the difference in the emoji name

jefersonbitencourtv commented:
Hi @rogeriofonseca 

This is what serves me today for emojis received by whatsapp, all emojis sent by whatsapp were tested working correctly, saving in the database as alias.

On the frontend we use the emojione lib to transform emojis into images on our platform and it works correctly

rogeriofonseca commented:
> Hi @rogeriofonseca
> 
> This is what serves me today for emojis received by whatsapp, all emojis sent by whatsapp were tested working correctly, saving in the database as alias.
> 
> On the frontend we use the emojione lib to transform emojis into images on our platform and it works correctly

I'm not sure about that. I'm just trying to help. 
When I run the unit tests I get the errors I said. When I submitted a contribution they ran the tests and showed another message.

Check it out.
https://github.com/vdurmont/emoji-java/pull/180#issuecomment-860085330
